### PR TITLE
fix/create_multiple_run_instances

### PR DESCRIPTION
**Pull request by davidebukali**
#217

### CODE REVIEW

File: core/serializers.py [hunk 1]
```diff
@@ -281,11 +285,27 @@ class Meta:
             'status'
         )
 
+    def isInstanceRunning(self, scheduled_event):
```
Comment (edwinmp) [inline core/serializers.py]: It's python ... they do snake case for functions and variables.

File: core/serializers.py [hunk 2]
```diff
@@ -281,11 +285,27 @@ class Meta:
...
+        queryset = ScheduledEventRunInstance.objects.filter(
+            (Q(scheduled_event=scheduled_event.id) & Q(status='r'))
+        )
+        if queryset.exists():
+            return queryset.earliest('id')
+
+    def isDueToRunIn5Minutes(self, scheduled_event):
```
Comment (edwinmp) [inline core/serializers.py]: Same here.

File: core/views.py [hunk 3]
```diff
@@ -418,12 +419,35 @@ def get(self, request, pk, format=None):
             serializer = ScheduledEventRunInstanceSerializer(scheduled_event_run_instance, many=True)
             return Response(serializer.data)
 
+    def getPostResponse(self, serializer, request):
```
Comment (edwinmp) [inline core/views.py]: And here...

**Review by edwinmp — CHANGES_REQUESTED** [review]

File: core/serializers.py (hunk 1 above)
Comment (davidebukali) [reply core/serializers.py]: Updated

File: core/serializers.py (hunk 2 above)
Comment (davidebukali) [reply core/serializers.py]: Updated

File: core/views.py (hunk 3 above)
Comment (davidebukali) [reply core/views.py]: Updated

**Review by edwinmp — APPROVED** [review]

**Pull request merged**